### ISSUE TITLE
Add Fedora URI as (psuedo?) field so it can be displayed / themed

### ISSUE THREAD

**Issue by dannylamb** [Islandora-CLAW/CLAW#881]
The Fedora URI should also be made available to site builders.  This will require a Gemini lookup after ingest.  It might be best to do this as a pseudo field so long as we can guarantee it will get cached.  Otherwise we'll have to stuff it in the database, but that's less than ideal.

**Comment by dannylamb** [comment]
As of https://github.com/Islandora-CLAW/Crayfish-Commons/releases/tag/0.0.2, you can `composer require islandora/crayfish-commons` to get a Gemini client.

**Comment by whikloj** [comment]
Ok I have moved through a computed field and landed right where @dannylamb said all along in a pseudo field.

I worked to integrate it into a context but what I realize is that from a configuration stand-point I just need a list of what entity types and bundles this field is allowed to be on. The actual configuration is done in the field display management.

So rather than using a context reaction and then trying to figure out what type of entity was allowed in the context condition. 

So would anyone mind if I just made a configuration form that listed all the media, node and file bundles and you check those that the form "can" be displayed on?

Whether it is displayed for a specific entity type and bundle would be (as I said) configurable elsewhere.
